src/storage/paths.ts

```diff
@@ -66,6 +66,19 @@ export function getCliPath(): string {
 		: path.join( getResourcesPath(), 'cli', 'main.js' );
 }
 
+export function getBundledNodeBinaryPath(): string {
+	const nodeBinaryName = process.platform === 'win32' ? 'node.exe' : 'node';
+
+	if ( process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test' ) {
```

Comment:
I am thinking - should we keep it only for tests? To have it used in development, to work with the same as in prod, to avoid if in prod something will be unexpectedly broken?
```suggestion
	if ( process.env.NODE_ENV === 'test' ) {
```